### ISSUE TITLE
windows 11 working?

### ISSUE THREAD

**Issue by l-urk** [x0xr00t/Automated-MUlti-UAC-Bypass#25]
22H2

**Comment by x0xr00t** [comment]
@l-urk Can you provide more details?, as far as i know it works on 22h2. 
Show me the output of powershell, also do check the following things. 

1.) check if tamper is disabled. 
2.) check if real time protection is temp off. 
3.) if for any reason this is not working after that ( include a screenshot from them ps terminal output, and also add the full version info pulled from ps1 (build number and such)

I can then look in to the issue. 
Kinds, 

x0xr00t

**Comment by l-urk** [comment]
@x0xr00t I don't remember why I started this thread exactly but... I've modified the script to the point where I'm able to actually not only open a shell with full token privilege (automated), but actually inject commands as that high privilege user. 
So basically insert an arbitrary payload after during the escalation (also hidden).
This can also be obfuscated.
Possibly you would be interested.. My discord is @l_kira___
